ext_emconf.php

```diff
@@ -2,4 +2,11 @@
 $EM_CONF['pti_twig'] = [
     'title' => 'PTI Twig',
     'description' => 'Prototype Integration - Twig',
+    'version' => '3.0',
```

Comment:
what happens if we leave the version number unset in ext_emconf? It would be nicer to do this via git tags only.